README.md

```diff
@@ -39,6 +40,94 @@ Note that politeiawww does not provide HTML output.  It strictly handles the
 JSON REST RPC commands only.  The GUI for politeiawww can be found at:
 https://github.com/decred/politeiagui
 
+## Development 
+
+#### 1. Install [Go](https://golang.org/doc/install) and [dep](https://github.com/golang/dep), if you haven't already.
+#### 2. Clone this repository. 
+#### 3. Setup configuration files: 
+* `politeiad` and `politeiawww` both have configuration files that you should set up to make execution easier. 
+* You should create the configuration files under the following path:  
+  - **Mac OS** `/Users/<username>/Library/Application Support/Politeiad/politeiad.conf`
+  - **Windows** `C:\Users\<username>\AppData\Local\Politeiad/politeiad.conf`
+  - **Ubuntu** `~/.politeiad/politeiad.conf`
+  
+* Copy and change the  [`sample-politeiawww.conf`](https://github.com/decred/politeia/blob/master/politeiawww/sample-politeiawww.conf) and [`sample-politeiad.conf`](https://github.com/decred/politeia/blob/master/politeiad/sample-politeiad.conf) files.
+* You may use the following default configurations:
+
+**politeiad.conf**: 
+`rpcuser=user rpcpass=pass testnet=true`	
```

Comment:
I think you should leave this indented as a code block, like you did underneath `politeiawww.cfg`.